README.md

```diff
@@ -90,25 +90,83 @@ Playwright MCP will launch the browser with the new profile, located at
 
 All the logged in information will be stored in that profile, you can delete it between sessions if you'd like to clear the offline state.
 
-### Running headless browser (Browser without GUI)
+### Configuration file
 
-This mode is useful for background or batch operations.
+The Playwright MCP server can be configured using a JSON configuration file. Here's the complete configuration format:
 
-```js
+```typescript
```

Comment:
I think having a JSON Schema would be appropriate here in the long-term.

Reply:
I don't think schema is appropriate in a readme, it is not readable.